coffee_cmd/src/main.rs

```diff
@@ -56,7 +56,29 @@ async fn main() -> Result<(), CoffeeError> {
             let remotes = coffee.list().await;
             coffee_term::show_list(remotes)
         }
-        CoffeeCommand::Upgrade => coffee.upgrade(&[""]).await,
+        CoffeeCommand::Upgrade { repo, all } => {
+            let result = match repo {
+                Some(repo) => coffee.upgrade(&repo, all).await,
+                None => coffee.upgrade("", all).await,
+            };
+            match result {
+                Ok(val) => {
+                    for res in val.total_status {
+                        match res.status {
+                            UpgradeStatus::UpToDate => {
+                                term::info!("Remote repository {} is already up to date!", res.repo)
+                            }
+                            UpgradeStatus::Updated => term::success!(
+                                "Remote repository {} was successfully upgraded!",
+                                res.repo
+                            ),
+                        }
+                    }
+                }
+                Err(err) => term::error(format!("{err}")),
+            }
+            Ok(())
```

Comment:
Mh, this upgrade is all creating problem, if one fails we should abort the upgrade, otherwise, we leave the job not completely done.

For now we do not allow upgrading a single repository by name only? what do you think?